entity-types/ext-cox_generic/definition.yml

```diff
@@ -0,0 +1,24 @@
+domain: EXT
+type: COX_GENERIC
+
+synthesis:
+  rules:
+  - identifier: appName
+    name: appName
+    encodeIdentifierInGUID: true
+    conditions:
+    - attribute: source
+      value: coxGenericSysHealthPulse
+    tags:
+      stack:
+      state:
```

Comment:
is this something that will be changing?
Like: running, paused, stoped, etc?

IF that's the case this will have multiple values and you may want only the latest one. If that's the case use multiValue: false